src/valkey-benchmark.c

```diff
@@ -364,6 +393,7 @@ static void freeClient(client c) {
         }
     }
     valkeyFree(c->context);
+    if (c->paused)  releasePausedClient(c);
```

Comment:
Clang-format complains about the extra space.

```suggestion
    if (c->paused) releasePausedClient(c);
```